app/src/main/java/com/spendster/presentation/signUp/SignUpActivity.java

```diff
@@ -10,34 +10,32 @@
 
 import com.spendster.BuildConfig;
 import com.spendster.R;
+import com.spendster.presentation.AuthView;
 import com.spendster.presentation.moreAboutYou.MoreAboutYouActivity;
 
-public class SignUpActivity extends AppCompatActivity implements SignUpView {
+public class SignUpActivity extends AppCompatActivity implements AuthView {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_sign_up);
 
-        final SignUpPresenter signUpPresenter = new SignUpPresenter(this, new ServerSignUpModel());
         final Button btnContinue = findViewById(R.id.btnContinue);
```

Comment:
@SimonMaha this code should be extracted from here, the initUI method would be a nice place to extract to.